### ISSUE TITLE
可以用，但是OC报错

### ISSUE THREAD

**Issue by star-liu** [Enmusushi/XiaomiAir13.3TM1604-EFI#1]
直接替换原来OC可以用，但是无法在开机进入OC选择微软系统或者MAC的界面
![IMG_20210601_153628948](https://user-images.githubusercontent.com/73949314/120355822-3896f680-c336-11eb-986c-7d9e0492a3bd.jpg)

**Comment by Enmusushi** [comment]
嗯，好吧，我是只装Mac系统，你这是想装双系统吧？

**Comment by vanyasem** [comment]
Sorry to hijack this old thread, but I've been running macOS on my **Air 2017 7200u** for the past 5 years, and I've recently decided to upload my configuration to GitHub: https://github.com/vanyasem/hackintosh-xiaomi-air-7200U

It is almost perfect for this particular laptop, and I still update it to this day.

One caveat though is that I focus on supporting one macOS version at a time to ensure maximum reliability (all changes are tested manually by me prior to releasing an update). The currently supported macOS version is mentioned in the readme.

**Comment by star-liu** [comment]
didn't support  Monterey ？Wi-Fi can not use on macOS12.1在 2024年2月14日，11:29，Ivan Semkin ***@***.***> 写道：﻿
Sorry to hijack this old thread, but I've been running macOS on my Air 2017 7200u for the past 5 years, and I've recently decided to upload my configuration to GitHub: https://github.com/vanyasem/hackintosh-xiaomi-air-7200U
It is almost perfect for this particular laptop, and I still update it to this day.
One caveat though is that I focus on supporting one macOS version at a time to ensure maximum reliability (all changes are tested manually by me prior to releasing an update). The currently supported macOS version is mentioned in the readme.

**Comment by vanyasem** [comment]
@star-liu no, it does not support Monterey. The currently supported version is macOS 13 Ventura

As long as the latest Xcode is still supported on Ventura I will not make the update to Sonoma

Official Kaby Lake support was dropped in Sonoma, and this laptop is based on Kaby Lake. And as I try to stick to the most native and patch-free experience, I don't see a point in updating to Sonoma until I am forced to do so by Xcode

Upd: Xcode 15.3 is only supported on Sonoma. I will make the update soon

**Comment by star-liu** [comment]
OK，thank you for sharing the EFI。I will update the macOS later。在 2024年2月19日，12:07，Ivan Semkin ***@***.***> 写道：﻿
@star-liu no, it does not support Monterey
As long as the latest Xcode is still supported on Ventura I will not make the update to Monterey
Official Kaby Lake support was dropped in Monterey, and this laptop is based on Kaby Lake. And as I try to stick to the most native and patch-free experience, I don't see a point in updating to Monterey until I am forced to do so by Xcode

**Comment by vanyasem** [comment]
@star-liu I have made an update that supports macOS Monterey for you: https://github.com/vanyasem/hackintosh-xiaomi-air-7200U/releases/tag/oc0.9.9_macos12-14

It supports macOS 12 - Monterey, macOS 13 - Ventura, macOS 14 - Sonoma (including the latest 14.4 version)

Enjoy!

**Comment by star-liu** [comment]
Wow , thanks! I have updated to Ventura.👍👍在 2024年3月13日，10:11，Ivan Semkin ***@***.***> 写道：﻿
@star-liu I have made an update that supports macOS Monterey for you: https://github.com/vanyasem/hackintosh-xiaomi-air-7200U/releases/tag/oc0.9.9_macos12-14
It supports macOS 12 - Monterey, macOS 13 - Ventura, macOS 14 - Sonoma (including the latest 14.4 version)
Enjoy!

**Comment by star-liu** [comment]
I'm a dual system of win and mac, and I can't find the win startup item when oc starts. How to add a win startup item?在 2024年3月13日，10:22，QQ Mail ***@***.***> 写道：﻿Wow , thanks! I have updated to Ventura.👍👍在 2024年3月13日，10:11，Ivan Semkin ***@***.***> 写道：﻿
@star-liu I have made an update that supports macOS Monterey for you: https://github.com/vanyasem/hackintosh-xiaomi-air-7200U/releases/tag/oc0.9.9_macos12-14
It supports macOS 12 - Monterey, macOS 13 - Ventura, macOS 14 - Sonoma (including the latest 14.4 version)
Enjoy!
